flw/src/main/resources/flw.properties

```diff
@@ -0,0 +1,7 @@
+#what time of day should the process of generating the targetFile start
```

Comment:
I was wondering why the discarding of old FLW should coincide with targetFile generation